homeassistant/components/image_processing/__init__.py

```diff
@@ -113,24 +114,24 @@ class ImageProcessingEntity(Entity):
     timeout = DEFAULT_TIMEOUT
 
     @property
-    def camera_entity(self):
+    def camera_entity(self) -> str | None:
         """Return camera entity id from process pictures."""
         return None
 
     @property
-    def confidence(self):
+    def confidence(self) -> float | None:
         """Return minimum confidence for do some things."""
         return None
 
-    def process_image(self, image):
+    def process_image(self, image: Image) -> None:
         """Process image."""
         raise NotImplementedError()
 
-    async def async_process_image(self, image):
+    async def async_process_image(self, image: Image) -> None:
```

Comment:
Shouldn't `image` be annotated as `bytes`?

In `async_update` `image.content` is passed to `async_process_image`.

https://github.com/home-assistant/core/blob/5501b7e710bed9ec32b2afc195a75f37eab1bf69/homeassistant/components/image_processing/__init__.py#L134-L152

Reply:
Indeed, fixed in #78426